bitnami/mariadb-galera/Chart.yaml

```
@@ -28,4 +28,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-mariadb-galera
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 5.5.1
+version: 5.5.2
```

Please bump the minor version instead since you're adding new parameters.
```suggestion
version: 5.6.0
```

---

thanks for the comment - I just did bump the minor version